src/utils/getAllFiles.ts

```diff
@@ -0,0 +1,27 @@
+import path from "path";
+import fs from "fs-extra";
+
+/**
+ * Find all files inside a dir, recursively.
+ * https://gist.github.com/kethinov/6658166?permalink_comment_id=3379782#gistcomment-3379782
+ * https://gist.github.com/kethinov/6658166?permalink_comment_id=2389484#gistcomment-2389484
+ * @function getAllFiles
+ * @param {string} dir Directory path string.
+ * @param {string} base Base path string.
+ * @return {string[]} Array with all the relative paths that are inside the directory.
+ */
```

Comment:
Not a fan of jsdoc and we don't use it anywhere else. Using typescript is clear enough and inlined. Leave the reference link if it is taken externally.